Makefile

```diff
@@ -32,7 +32,11 @@ docker/build:
 
 .PHONY: run
 run:
-	@reflex -c reflex.conf -- sh -c ./bin/Will.IAM start-api
+	make build && ./bin/Will.IAM start-api --host=localhost -v3
```

Comment:
Without adding `--host=localhost` command, every time we compile and run Will.IAM we receive a MacOS Firewall message box asking us to allow connections to this app. 
Telling that you are using `localhost` avoid it on development.